pontoon/base/static/js/fluent_interface.js

```diff
@@ -670,6 +678,13 @@ var Pontoon = (function (my) {
           $('#ftl-area textarea.value').keyup();
         }
 
+        // Indent selector variants if preceeded by other elements
+        $('.ftl-area section > ul:not(.template) > li[data-expression]').each(function(i, node) {
+          if ($(node).prevAll(':visible:not([data-expression])').length) {
+            $(node).addClass('indented');
+          }
+        });
```

Comment:
Actually, I'm wondering if this is the right solution. Why did you decide to add that class afterwards using selectors rather than during the creation of the elements?

Reply:
Yep, I figured there was a complexity issue involved, just wanted to make sure. That's fine by me!